src/components/Dashboard/ThisWeek/NewsletterMenu/MenuItems/ExportPhysical/ExportPhysical.test.tsx

```diff
@@ -70,15 +97,96 @@ describe('ExportPhysical', () => {
       window.location = location;
     });
 
-    it('Exports Contacts and Downloads File', async () => {
-      const url = 'someRandomUrlToFile';
-      const mocks = {
-        CreateExportedContacts: {
-          exportContacts: url,
-        },
-      };
+    it('Exports Contacts and Downloads File - PDF of Mail Merged Labels | Avery5160 and Contact Name', async () => {
+      const { getByText, getByTestId } = render(
+        <GqlMockedProvider<CreateExportedContactsMutation>
+          mocks={createMock(ExportFormatEnum.Pdf)}
+        >
+          <ExportPhysical
+            accountListId={accountListId}
+            handleClose={handleClose}
+          />
+        </GqlMockedProvider>,
+      );
+      userEvent.click(getByText('PDF of Mail Merged Labels'));
+      // value of the select is held in a child Input element, so we must reference that element for the value
+      expect(getByTestId('label-template-select').childNodes[1]).toHaveValue(
+        ExportLabelTypeEnum.Avery5160,
+      );
+      expect(getByTestId('sort-by-select').childNodes[1]).toHaveValue(
+        ExportSortEnum.Name,
+      );
+
+      userEvent.click(getByText('Export'));
+      await waitFor(() =>
+        expect(window.location.replace).toHaveBeenCalledWith(
+          `${url}.${ExportFormatEnum.Pdf}?access_token=${token}`,
+        ),
+      );
+    });
+
+    it('Exports Contacts and Downloads File - PDF of Mail Merged Labels | Avery7160 and Zip', async () => {
+      const { getByText, getByTestId, getByLabelText, getByRole } = render(
+        <GqlMockedProvider<CreateExportedContactsMutation>
+          mocks={createMock(ExportFormatEnum.Pdf)}
+        >
+          <ExportPhysical
+            accountListId={accountListId}
+            handleClose={handleClose}
+          />
+        </GqlMockedProvider>,
+      );
+
+      userEvent.click(getByText('PDF of Mail Merged Labels'));
+      //  material ui selects do not use actual Select elements in the DOM
+      //  reference https://stackoverflow.com/a/61491607
```

Comment:
Having to test the selects this way is a bit annoying but glad I was able to find this solution.